elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -35,12 +35,6 @@ li.ui-state-highlight {
   min-width: 3em !important;
 }
 
-ul.header {
-  color: black;
-  list-style-type: none;
-  margin-bottom: 0px;
-}
-
 ul.inline li {
```

Comment:
It's a big red flag for these classes to not be scoped to this element, as they'd impact everything on the page. Can you change this to be scoped to an element-specific class like the other selectors? Ditto with everything else in this file.